dll/win32/newdev/newdev.c

```diff
@@ -353,6 +353,10 @@ SearchDriverRecursive(
     BOOL retval = FALSE;
     HANDLE hFindFile = INVALID_HANDLE_VALUE;
 
+    if (wcslen(Path) + 3 /* pattern + zero terminal */ > MAX_PATH)
+        /* Path is too long to be searched */
```

Comment:
Please reformat the comments. We don't put comments in the same line with the code this way. Our [Coding Style](https://reactos.org/wiki/Coding_Style) will help you sorting out though this particular case isn't covered.